### PR TITLE
Display specefic message when no limit violation is found

### PR DESCRIPTION
**Pull request by AAJELLAL**

### CODE REVIEW

File: src/components/loadflow-result.js [hunk 1]
```diff
@@ -73,7 +75,25 @@ const LoadFlowResult = ({ result, studyUuid, nodeUuid }) => {
     const limitReductionParam = useSelector((state) =>
         Number(state[PARAM_LIMIT_REDUCTION])
     );
-    const loadflowNotif = useSelector((state) => state.loadflowNotif);
+    const loadFlowStatus = useSelector(
+        (state) => state.computingStatus[ComputingType.LOADFLOW]
+    );
+    const messages = useMemo(() => {
```
[the remark this answers is not in the document]
Comment (AAJELLAL) [reply src/components/loadflow-result.js]: Done.

File: src/components/results/sensitivity-analysis/sensitivity-analysis-result.js [hunk 2]
```diff
@@ -37,7 +40,25 @@ const SensitivityAnalysisResult = ({
 }) => {
     const gridRef = useRef(null);
     const intl = useIntl();
-
+    const sensitivityAnalysisStatus = useSelector(
+        (state) => state.computingStatus[ComputingType.SENSITIVITY_ANALYSIS]
+    );
+    const messages = useMemo(() => {
```
[the remark this answers is not in the document]
Comment (AAJELLAL) [reply src/components/results/sensitivity-analysis/sensitivity-analysis-result.js]: Done

File: src/components/results/shortcircuit/shortcircuit-analysis-result-table.tsx [hunk 3]
```diff
@@ -108,7 +112,27 @@ const ShortCircuitAnalysisResult: FunctionComponent<
             },
         ];
     }, [intl]);
+    const shortCircuitAnalysisStatus = useSelector(
+        (state: ReduxState) =>
+            state.computingStatus[ComputingType.SHORTCIRCUIT_ANALYSIS]
+    );
 
+    const messages = useMemo(() => {
```
[the remark this answers is not in the document]
Comment (AAJELLAL) [reply src/components/results/shortcircuit/shortcircuit-analysis-result-table.tsx]: Done

File: src/components/security-analysis-result.js [hunk 4]
```diff
@@ -55,8 +57,26 @@ const SecurityAnalysisResult = ({ onClickNmKConstraint, result }) => {
 
     const intl = useIntl();
 
-    const saNotif = useSelector((state) => state.saNotif);
+    const securityAnalysisStatus = useSelector(
+        (state) => state.computingStatus[ComputingType.SECURITY_ANALYSIS]
+    );
 
+    const messages = useMemo(() => {
```
[the remark this answers is not in the document]
Comment (AAJELLAL) [reply src/components/security-analysis-result.js]: Done

**Review by trikimahdi — APPROVED** [review]
Test ok
Code ok

**Pull request merged**